static/js/util/test_utils.js

```diff
@@ -21,14 +22,38 @@ export function findCourse(courseSelector: (course: Course, program: Program) =>
   throw "Unable to find course";
 }
 
-export const alterFirstRun = (course: Course, overrideObject: Object): Course => {
+export const alterFirstRun = (course: Course, overrideObject: Object): CourseRun => {
   course.runs[0] = Object.assign({}, course.runs[0], overrideObject);
+  return course.runs[0];
 };
 
 export function findAndCloneCourse(courseSelector: (course: Course, program: Program) => boolean): Course {
   return _.cloneDeep(findCourse(courseSelector));
 }
 
+export function setCourseRunCount(course: Course, desiredRunCount: number, courseRunToCopy: ?CourseRun) {
+  let currentRunCount = course.runs.length;
+  if (currentRunCount < desiredRunCount) {
+    let courseRun = currentRunCount === 0 ? courseRunToCopy : course.runs[0];
+    if (!courseRun) {
+      throw new Error('Need a course run to copy.');
+    }
+    let runsNeeded = desiredRunCount - currentRunCount;
+    let idMax = _.max(_.map(course.runs, run => run.id)) || 0;
```

Comment:
the point of this function is to generate data for a single Course. it's not concerned with the structure of the entire response data. i don't quite understand why this function, or the tests that use it, need to worry about that. let's chat if you want to explain further